.github/CODEOWNERS

```
@@ -309,6 +309,7 @@
 /core/templates/services/math-interactions.service.ts @iamprayush
 /extensions/domain*.py @brianrodri
 /extensions/interactions/ @iamprayush @vojtechjelinek
+/extensions/objects/templates/filepath-editor.component.* @kevintab95
```

```suggestion
/extensions/objects/templates/filepath-editor.component*.ts @kevintab95
```

Following the existing patterrns

---

Done.